src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/php/workflows.php

```diff
@@ -14,12 +14,19 @@
             ),
             'initial_place' => 'draft',
             'places' => array(
-                'draft',
-                'wait_for_journalist',
-                'approved_by_journalist',
-                'wait_for_spellchecker',
-                'approved_by_spellchecker',
-                'published',
+                'draft' => array(),
+                'wait_for_journalist' => array(
+                    'metadata' => array(
+                        'dump_style' => array(
```

Comment:
the extra level is not really needed is it. We could reserve 1st level keywords (label, color, description) for any generic purpose.